api/api.go

```diff
@@ -164,3 +176,92 @@ func OrbPublish(ctx context.Context, logger *logger.Logger,
 	}
 	return &response.PublishOrb.PublishOrbResponse, err
 }
+
+func createNamespaceById(ctx context.Context, logger *logger.Logger, name string, ownerId string) (*CreateNamespaceResponse, error) {
+	var response struct {
+		CreateNamespace struct {
+			CreateNamespaceResponse
+		}
+	}
+
+	query := `
+			mutation($name: String!, $organizationId: UUID!) {
+				createNamespace(
+					name: $name,
+					organizationId: $organizationId
+				) {
+					namespace {
+						createdAt
+						id
+					}
+					errors {
+						message
+						type					
+					}
+				}
+			}`
+
+	request := client.NewAuthorizedRequest(viper.GetString("token"), query)
+	request.Var("name", name)
+	request.Var("organizationId", ownerId) 	// TODO: In GraphQL, organizationId should really be ownerId
+
+	graphQLclient := client.NewClient(viper.GetString("endpoint"), logger)
+
+	err := graphQLclient.Run(ctx, request, &response)
+
+	if err != nil {
+		err = errors.Wrap(err, fmt.Sprintf("Unable to create namespace %s for ownerId %s", name, ownerId))
+	}
+
+	return &response.CreateNamespace.CreateNamespaceResponse, err
+}
+
+func GetOrganization(ctx context.Context, logger *logger.Logger, organizationName string, organizationVcs string) (string, error){
+	var response struct {
+		Organization struct {
+			Id string
+		}
+	}
+
+	query := `
+			query($organizationName: String!, $organizationVcs: VCSType!) {
+				organization(
+					name: $organizationName
+					vcsType: $organizationVcs
+				) {
+					id
+				}
+			}`
+
+
+	request := client.NewAuthorizedRequest(viper.GetString("token"), query)
+	request.Var("organizationName", organizationName)
+	request.Var("organizationVcs", organizationVcs)
+
+	graphQLclient := client.NewClient(viper.GetString("endpoint"), logger)
+
+	err := graphQLclient.Run(ctx, request, &response)
+
+	fmt.Println(response)
```

Comment:
I think you want to delete this line before merging? (In my editor, doing this would also get rid of the "fmt" import at the top on save.  I think a default go tool does that, but not sure which.)